src/openms/source/DATASTRUCTURES/Param.cpp

```diff
@@ -1222,7 +1222,7 @@ OPENMS_THREAD_CRITICAL(oms_log)
             }
             this->root_.insert(local_entry, prefix); //->setValue(it.getName(), local_entry.value, local_entry.description, local_entry.tags);
           }
-          else
+          else if(verbose)
```

Comment:
so this is now silenced? might be a bit dangerous

Reply:
if you do 'verbose == true' then its active again :)
But you mean for other use cases like `-write_ini + -ini` ?

Reply:
yep. Is it active by default if one uses an old ini file in a new version?